### DISCUSSION TITLE
Exception training model: ''AttnProcessor2_0' object has no attribute 'state_dict''.

### DISCUSSION THREAD

**Discussion by DrozhzhinDen** [d8ahazard/sd_dreambooth_extension#1364] [General]
Friends, help! After completing all the steps of the training, at the moment of saving the model, it throws out "Exception training model: ''AttnProcessor2_0' object has no attribute 'state_dict''.".

This is what is happening in the console:
AttributeError: 'AttnProcessor2_0' object has no attribute 'state_dict'
Steps: 100%|█████████████████████████████████████| 294/294 [02:13<00:00,  2.20it/s, loss=0.0746, lr=1.26e-6, vram=12.5]
Duration: 00:02:18
Saving Lora Weights...:   0%|                                                                    | 0/1 [00:00<?, ?it/s]
Duration: 00:02:19

**Comment by 1099271** [comment]
same to me!

**Comment by ninofiliu** [comment]
This is being tracked as a bug https://github.com/d8ahazard/sd_dreambooth_extension/issues/1363

**Comment by EstherLee1995** [comment]
Have you solved it out? Same here

**Comment by 553672759** [comment]
in save tab ,You can try checking the box  Save Checkpoint to Subdirectory   and  Generate a .ckpt file when saving during training.
im trying it.
